datasets/gov/rrt/el_rysiu_ir_pasto_teikejai.csv

```diff
@@ -0,0 +1,22 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/rrt/el_rysiu_ir_pasto_teikejai,,,,,,,,,,,,Elektroninių ryšių bei pašto paslaugų teikėjai ir jų pateiktų periodinių ataskaitų statistika,
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,,,,,,,,
+,,,,ElRysiaiPastasTeikejas,,,vda_prime_key,,,4,,,Elektroninių ryšių bei pašto paslaugų teikėjai ir jų pateiktų periodinių ataskaitų statistika,"""Lentelėje pateikiama Lietuvos Respublikos ryšių reguliavimo tarnybai pateiktų pašto paslaugų ir elektroninių ryšių ataskaitų statistika. Duomenys pateikiami nuo 2021 metų. Metai ir ketvirtis nurodo periodą, už kurį pateikta ataskaita, bet ne ataskaitos pateikimo laiką. Vienam ketvirčiui yra pateikiama viena ataskaita. Jei įmonė tuo pačiu metu teikia ir pašto, ir elektroninių ryšių paslaugas ji gali pateikti dvi skirtingas ataskaitas tam pačiam ketvirčiui. Duomenys atnaujinami kas 3 mėnesius."""
+,,,,,vda_prime_key,string,,,,4,open,,Duomenų eilutės įrašo identifikatorius,
+,,,,,metai,integer,,,,4,open,,"""Metai, už kuriuos pateikta ataskaita""",
+,,,,,ketvirtis,string,,,,4,open,,"""Ketvirtis, už kurį pateikta ataskaita""",
+,,,,,,enum,,,"""I""",,,,Pirmas ketvirtis,
+,,,,,,,,,"""II""",,,,Antras ketvirtis,
+,,,,,,,,,"""III""",,,,Trečias ketvirtis,
+,,,,,,,,,"""IV""",,,,Ketvirtas ketvirtis,
+,,,,,ataskaitos_tipas,string,,,,4,open,,Ataskaitos tipas,
+,,,,,,enum,,,"""Pašto paslaugos ataskaita""",,,,Pašto paslaugos ataskaita,
+,,,,,,,,,"""Elektroninių ryšių ataskaita""",,,,Elektroninių ryšių ataskaita,
+,,,,,imones_pavadinimas,string,,,,4,open,,Įmonės pavadinimas,
+,,,,,jar_kodas,string,,,,4,open,,Juridinių asmenų registro kodas,

```

Comment:
```suggestion
,,,,,jar_kodas,ref,/datasets/gov/rc/jar/iregistruoti/JuridinisAsmuo[ja_kodas],,,4,open,,Juridinių asmenų registro kodas,
```
Jei įmanoma, tai reiktų susieti ref su RC duomenu rinkiniu (https://get.data.gov.lt/datasets/gov/rc/jar/iregistruoti/JuridinisAsmuo)

Reply:
Yra ir užsienietiškų jar kodų. Pridėsiu į description šitą info.